Library/Homebrew/gpg.rb

```diff
@@ -38,12 +37,21 @@ def self.create_test_key(path)
       Key-Length: 2048
       Subkey-Type: RSA
       Subkey-Length: 2048
-      Passphrase: ''
```

Comment:
Think this was just added; does `%no-protection` do the same thing?

Reply:
Aye.